addons/web/static/src/views/fields/binary/binary_field.xml

```diff
@@ -16,7 +16,6 @@
                                 aria-label="Download"
                                 t-on-click="onFileDownload"
                             >
-                                <i class="fa fa-fw fa-download"/>
```

Comment:
Then the template is broken in form views.
Before (see the icons on the bottom right)
![image](https://github.com/odoo/odoo/assets/10908305/307701aa-3bee-4104-9887-5d348e5c5309)

After
![image](https://github.com/odoo/odoo/assets/10908305/be61617e-4c36-4258-908f-67f6a5299faa)

I think the override is wrong, not the original template

https://github.com/odoo/odoo/pull/129631/files#diff-b7ff2e5a9042fc4e88e7f54f7aef18f56c8669662c877b3ef28fa7356a83c241R64

Reply:
Definitely makes sense to keep the original template intact :+1: 
Pushed the changes. Thanks!